docs/docs/domain.mdx

```diff
@@ -154,26 +197,21 @@ Make this a nice table instead.
         type: bool
   ```
 
-
-
 * **Description**
 
-  Sets two boolean features. The first feature is `1` if the slot is set
-  and `0` otherwise. The second feature is `1` if the slot value is
-  `"true"`, `"1"`, or a variation of these, and `0` otherwise.
-
+  If `influence_conversation` is set to `true`, the assistant's behavior will change
+  depending on whether the slot is set to `true` or `false`. An empty `bool` slot
+  is the same as if the slot was set to `False`.
```

Comment:
> is the same as if the slot was set to `False`.

I find this surprising. does this have to be the case? this means a bool slot's default value is always `false`

Reply:
you can change the `initial_value`  🤔 I think changing this would also be out of the scope of this PR